hackathon_site/dashboard/frontend/src/testing/helpers.js

```diff
@@ -7,3 +7,9 @@ import store from "slices/store";
 export const withRouter = (component) => <BrowserRouter>{component}</BrowserRouter>;
 
 export const withStore = (component) => <Provider store={store}>{component}</Provider>;
+
+export const withStoreAndRouter = (component) => (
```

Comment:
Would flip the order of the internal components arround, currently it's acting as `withRouterAndStore` I think semantically. Or just have it call `withStore(withRouter(component))` directly.